WordPress/twentytwentyone
# Added dark theme preference

Fixes https://github.com/WordPress/twentytwentyone/issues/410

## Summary


This PR will add the dark mode preference to the theme

## Relevant technical choices:


## Test instructions

This PR can be tested by following these steps:
1. Set the OS color preference to dark.
1. Check if the default color is selected in the customizer that the theme is dark.


## Quality assurance
